### PR TITLE
E2E testsuite refactor for easier containerization

### PR DESCRIPTION
**Pull request by mmuzikar**
Unify the 2 testsuites used for testing each runtime into one independent of the runtime.
Introduced ability to test applications running in a docker container and to specify a URL of an already running application (both used for E2E tests).

Changed the workflows to run e2e tests on Java 11/17 x firefox/chrome x springboot/quarkus.

Container builds are done by maven plugin. The hawtio-test-suite image contains both chrome and firefox browsers to be able to use both for tests. Quarkus and Springboot apps are just plain containers with the app.

The e2e tests are now enabled only when using -Pe2e profile as the results were skipped by default previously, and specifying -Pe2e is not required in the tests/ directory.

I tried documenting the changes in tests/README but let me know if I missed something.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -45,12 +45,12 @@ mvn install -DskipTests
 ### Spring Boot 2.x E2E tests
 
 ```
-mvn install -Pe2e-springboot -am -pl tests/springboot
+mvn install -Pe2e -Pe2e-springboot -am -pl :hawtio-test-suite -Dlocal-app=true
```
Comment (tadayosi) [inline tests/README.md:48]: Nitpicky but comma (`-Pe2e,e2e-springboot`) should be a bit conciser.

File: tests/README.md [hunk 2]
```diff
@@ -45,12 +45,12 @@ mvn install -DskipTests
...
 ```
 
 ### Quarkus 2.x E2E tests
 ```
-mvn install -Pe2e-quarkus -am -pl tests/quarkus
+mvn install -Pe2e -Pe2e-quarkus -am -pl :hawtio-test-suite -Dlocal-app=true
```
Comment (tadayosi) [inline tests/README.md:53]: same here.

**Review by tadayosi — APPROVED** [review]
Looks great!

**Comment by tadayosi** [comment]
@mmuzikar E2E test runs are failing. Can you see why they fail?

**Comment by mmuzikar** [comment]
@tadayosi there's stability issue with chrome that I'm trying to figure out.

**Pull request merged**